packages/api/test/maintenance.spec.js

```diff
@@ -62,7 +62,7 @@ describe('maintenance middleware', () => {
     invalidModes.forEach((m) => {
       assert.throws(() => block(() => {}, {
         MODE: m
-      }, /invalid maintenance mode/))
+      }), /invalid maintenance mode/)
```

Comment:
yikes! a fine example of "type checking is good, actually".

Reply:
@olizilla haha yeah, not to mention all the `assert(` I changed to `assert.equal(`. I think the former wasn't actually asserting what it intended (truthiness vs equality)